app/containers/Send/Send.jsx

```diff
@@ -150,7 +150,7 @@ export default class Send extends React.Component<Props, State> {
     this.setState((prevState: Object) => {
       const newState = [...prevState.sendRowDetails]
 
-      if (newState.length < 5) {
+      if (newState.length < 10) {
```

Comment:
maybe 10 should be a variable that you can reuse throughout. This is the number of rows or recipients or something, right? What if we want to change it to 12 or 15. It would be easier if it was only defined in one place

Reply:
👍